change/@fluentui-react-2021-01-22-14-42-54-sareiff-forcedcolorsupdate.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
All these should be prerelease since the changes affect the published packages.

Reply:
Thanks! I was following this wiki (https://github.com/microsoft/fluentui/wiki/Change-Files) which uses different terminology so I wasn't sure what to choose

Reply:
Oh thanks for pointing that out. I'll update that page to mention prerelease.